Automattic/wp-calypso
# Use correct case on JSX components to avoid errors

#### Proposed Changes

When copying SVG elements into React sometimes we forget to convert kebab case attributes to camel case e.g. `fill-rule` should be converted to `fillRule`. Another common mistake is to use `class` instead of `className`. Even though these attributes get applied to the DOM correctly, React outputs an error in dev mode which pollutes the log and makes it harder to spot more serious errors. 

#### Testing Instructions

Should be enough to review the changes one by one. Functionally nothing has changed except the errors mentioned above are now gone. Before this change you would see a message in the log like this:

![image](https://user-images.githubusercontent.com/6851384/175937485-f8af6d65-df69-446f-a5c6-930913e85c0a.png)

Fixes #60677
